components/home/home.component.tsx

```diff
@@ -0,0 +1,96 @@
+import * as React from 'react';
+import { withFormik, FormikProps, FormikErrors, Form, Field } from 'formik';
+import * as Yup from 'yup';
+import Input from '../input-field/input';
+
+interface FormValues {
+  email: string;
+  password: string;
+}
+
+interface OtherProps {
+  text?: string;
+}
+
+interface MyFormProps {
+  initialEmail?: string;
+  initialPassword?: string;
+}
+
+const InnerForm = (props: OtherProps & FormikProps<FormValues>) => {
```

Comment:
Why do you need OtherProps?

```
const InnerForm: React.FC<FormikProps<FormValues>> = () => {}
```

Reply:
other props are needed to access onChange, onBlur and all other functionality need to be passed in input field of the formik

Reply:
But the type of OtherProps are:

```
type OtherProps = {
   text?: string;
 };
```

This does not match to what you are saying.

Reply:
ok, i'll look into it